### DISCUSSION TITLE
Are there CSS modules support?

### DISCUSSION THREAD

**Discussion by garkin** [alephjs/aleph.js#553] [Q&A] [answered]
```js
/* ./button.jsx */
import React, { Component } from 'react';
import styles from './button.module.css'; // Import css modules stylesheet as styles
import './another-stylesheet.css'; // Import regular stylesheet

class Button extends Component {
  render() {
    return <button className={styles.error}>Error Button</button>;
  }
}
```
```css
/* ./button.module.css */
.error {  background-color: red;  }
```
Doesn't looks like working with Aleph as of now. 
I see some weird example in git history
https://github.com/alephjs/aleph.js/tree/2a820b75daf75d4988c0b9730e9f37ebf30194a4/examples/hello-world-css-modules
Which doesn't look like having much in common with CSS modules and CSS component isolation at all.
And this example is also missing from current `main` branch.

**Comment by ije** [comment, marked as answer]
aleph uses lightning-css as the CSS pre-processor which supports css modules https://lightningcss.dev/css-modules.html, but i didn't add it as the feature yet
